tests/jax/test_jax_2d_eldry.py

```diff
@@ -636,8 +637,122 @@ def test_static_eldry(self):
             
             self.assertFalse(grad_failed, 
                          'Incorrect Gradient for static index {}'.format(i))
-        
         pass
 
+        
+    def test_eldry_test_7elem(self):

```

Comment:
In future, should have a comment at the top with triple quotes with a brief description of tests. Consistency for test comments is not perfect in general now, so this is okay.